### PR TITLE
windows/*: fix placeholders, flags and code descriptions (PR 1)

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

## ToDo

- [x] `assoc`
- [x] `attrib`
- [x] `azcopy`
- [x] `cd`
- [x] `chkdsk`
- [x] `choco-apikey`
- [x] `choco-feature`
- [x] `choco-info`
- [x] `choco-install`
- [x] `choco-list`
- [x] `choco`
- [x] `choco-new`
- [x] `choco-outdated`
- [x] `choco-pack`
- [x] `choco-pin`
- [x] `choco-search`
- [x] `choco-source`
- [x] `choco-uninstall`
- [x] `choco-upgrade`
- [x] `ftype`

### Next PRs

- [ ] `fc`
- [ ] `find`
- [ ] `findstr`
- [ ] `finger`
- [ ] `fondue`
- [ ] `forfiles`
- [ ] `for`
- [ ] `fsutil`
- [ ] `ftp`
- [ ] `get-childitem`
----
- [ ] `get-content`
- [ ] `get-filehash`
- [ ] `get-history`
- [ ] `getmac`
- [ ] `gpupdate`
- [ ] `if`
- [ ] `invoke-item`
- [ ] `ipconfig`
- [ ] `iscc`
- [ ] `logoff`
----
- [ ] `mkdir`
- [ ] `mklink`
- [ ] `more`
- [ ] `mount`
- [ ] `msg`
- [ ] `msiexec`
- [ ] `nfsstat`
- [ ] `nvm`
- [ ] `octo`
- [ ] `pabcnetcclear`
----
- [ ] `path`
- [ ] `pathping`
- [ ] `pipwin`
- [ ] `popd`
- [ ] `powershell`
- [ ] `print`
- [ ] `psping`
- [ ] `pushd`
- [ ] `pwlauncher`
- [ ] `query`
- [ ] `rdpsign`
----
- [ ] `reg-add`
- [ ] `reg-compare`
- [ ] `reg-copy`
- [ ] `reg-delete`
- [ ] `reg-export`
- [ ] `reg-flags`
- [ ] `reg-import`
- [ ] `reg-load`
- [ ] `reg`
- [ ] `reg-query`
----
- [ ] `reg-restore`
- [ ] `reg-save`
- [ ] `reg-unload`
- [ ] `repair-bde`
- [ ] `replace`
- [ ] `rmdir`
- [ ] `robocopy`
- [ ] `rpcinfo`
- [ ] `scoop-bucket`
- [ ] `scoop`
----
- [ ] `set`
- [ ] `setx`
- [ ] `sfc`
- [ ] `showmount`
- [ ] `shutdown`
- [ ] `subst`
- [ ] `systeminfo`
- [ ] `takeown`
- [ ] `taskkill`
- [ ] `tasklist`
----
- [ ] `time`
- [ ] `title`
- [ ] `tree`
- [ ] `tskill`
- [ ] `type`
- [ ] `tzutil`
- [ ] `vcvarsall`
- [ ] `ver`
- [ ] `virtualboxvm`
----
- [ ] `vol`
- [ ] `vswhere`
- [ ] `where`
- [ ] `whoami`
- [ ] `winget`
- [ ] `wmic`
- [ ] `wsl`
- [ ] `wsl-open`
- [ ] `xcopy`
- [ ] `explorer`

----
[Next PR][next]

[next]: https://github.com/tldr-pages/tldr/pull/7894

### CODE REVIEW

File: pages/windows/choco-new.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 # choco new
 
 > Generate new package specification files with Chocolatey.
-> More information: <https://chocolatey.org/docs/commands-new>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/new>.
 
-- Create a new package skeleton:
+- Create the new package skeleton:
```
Comment (blueskyson) [inline pages/windows/choco-new.md:6]: I'm sure that here should be "a" rather than "the" because it doesn't matter which package skeleton one creates.

**Review by blueskyson — APPROVED** [review]
All modifications looks fine except replacing "a" with "the". Why do you want to replace almost all "a" with "the"?

File: pages/windows/assoc.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-- Display the associated filetype for a specific extension:
+- Print the associated filetype for the specified extension (.txt):
 
 `assoc {{.txt}}`
 
-- Modify the associated filetype for a specific extension:
+- Set/remove the associated filetype (txtfile) for the specified extension (.txt):
 
-`assoc {{.txt}}={{txtfile}}`
+`assoc {{.txt}}={{txtfile| }}`
```
Comment (navarroaxel) [inline pages/windows/assoc.md:17]: What's this?
```suggestion
`assoc {{.txt}}={{txtfile}}`
```

File: pages/windows/azcopy.md [hunk 3]
```diff
@@ -7,22 +7,22 @@
...
 
-`azcopy copy '{{path/to/source}}' 'https://{{storage_account_name}}.blob.core.windows.net/{{container_name}}' --include-pattern '{{*.txt;*.jpg}}'`
+`azcopy copy {{path/to/source}} 'https://{{storage_account_name}}.blob.core.windows.net/{{container_name}}' --include-pattern '{{*.txt;*.jpg}}'`
 
 - Copy a container directly between two Azure storage accounts:
 
 `azcopy copy 'https://{{source_storage_account_name}}.blob.core.windows.net/{{container_name}}' 'https://{{destination_storage_account_name}}.blob.core.windows.net/{{container_name}}'`
 
-- Synchronize a local directory and delete files in the destination if they no longer exist in the source:
+- Synchronize a local directory and delete/preserve files in the destination if they no longer exist in the source:
```
Comment (navarroaxel) [inline pages/windows/azcopy.md:22]: I think the `delete/preserve` verbs don't have a clear relation with the `true|false` flag.

File: pages/windows/choco-apikey.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 
 `choco apikey`
 
-- Display a specific source and its API key:
+- Print the specified source and its API key:
 
 `choco apikey --source "{{source_url}}"`
 
-- Set an API key for a source:
+- Set the API key for the specified source:
```
Comment (navarroaxel) [inline pages/windows/choco-apikey.md:14]: `an` sounds better
```suggestion
- Set an API key for the specified source:
```

File: pages/windows/choco-info.md [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
+> More information: <https://docs.chocolatey.org/en-us/choco/commands/info>.
 
-- Display information on a specific package:
+- Print information for the specified package:
 
-`choco info {{package}}`
+`choco info {{package_name}}`
 
-- Display information for a local package only:
+- Print information for the specified local package:
```
Comment (navarroaxel) [inline pages/windows/choco-info.md:10]:
```suggestion
- Print information for a specified local package:
```

File: pages/windows/choco-info.md [hunk 6]
```diff
@@ -1,20 +1,20 @@
...
+`choco info {{package_name}} --local-only`
 
-- Specify a custom source to receive packages information from:
+- Get package information from the specified source:
 
-`choco info {{package}} --source {{source_url|alias}}`
+`choco info {{package_name}} --source {{source_url|source_alias}}`
 
-- Provide a username and password for authentication:
+- Provide the username and the password for authentication:
```
Comment (navarroaxel) [inline pages/windows/choco-info.md:18]:
```suggestion
- Provide a specific username and password for authentication:
```

File: pages/windows/choco-install.md [hunk 7]
```diff
@@ -1,36 +1,36 @@
...
+`choco install {{path/to/file.nuspec}}`
 
-- Specify a custom source to receive packages from:
+- Install package from the specified source:
 
-`choco install {{package}} --source {{source_url|alias}}`
+`choco install {{package_name}} --source {{source_url|source_alias}}`
 
-- Provide a username and password for authentication:
+- Provide the username and the password for authentication:
```
Comment (navarroaxel) [inline pages/windows/choco-install.md:34]:
```suggestion
- Provide a specific username and password for authentication:
```

File: pages/windows/choco-list.md [hunk 8]
```diff
@@ -1,28 +1,28 @@
...
+`choco list --local-only --include-programs`
 
-- Specify a custom source to display packages from:
+- Print all packages from the specified source matching the given pattern:
 
-`choco list --source {{source_url|alias}}`
+`choco list --source {{source_url|source_alias}}`
 
-- Provide a username and password for authentication:
+- Provide the username and the password for authentication:
```
Comment (navarroaxel) [inline pages/windows/choco-list.md:26]:
```suggestion
- Provide a specific username and password for authentication:
```

File: pages/windows/choco-new.md [hunk 9]
```diff
@@ -1,24 +1,24 @@
...
 
 `choco new {{package_name}} --maintainer {{maintainer_name}}`
 
-- Create a new package in a custom output directory:
+- Create the new package in the specified output directory:
 
 `choco new {{package_name}} --output-directory {{path/to/directory}}`
 
-- Create a new package with specific 32-bit and 64-bit installer URLs:
+- Create the new package with specific 32-bit and 64-bit installer URLs:
```
Comment (navarroaxel) [inline pages/windows/choco-new.md:22]:
```suggestion
- Create a new package with specific 32-bit and 64-bit installer URLs:
```

File: pages/windows/choco-new.md [hunk 10]
```diff
@@ -1,24 +1,24 @@
...
-`choco new {{package_name}} --version {{version}}`
+`choco new {{package_name}} --version {{package_version}}`
 
-- Create a new package with a specific maintainer name:
+- Create the new package with the specified maintainer name:
 
 `choco new {{package_name}} --maintainer {{maintainer_name}}`
 
-- Create a new package in a custom output directory:
+- Create the new package in the specified output directory:
```
Comment (navarroaxel) [inline pages/windows/choco-new.md:18]:
```suggestion
- Create a new package in the specified output directory:
```

File: pages/windows/choco-new.md [hunk 11]
```diff
@@ -1,24 +1,24 @@
...
 `choco new {{package_name}}`
 
-- Create a new package with a specific version:
+- Create the new package with the specified version:
 
-`choco new {{package_name}} --version {{version}}`
+`choco new {{package_name}} --version {{package_version}}`
 
-- Create a new package with a specific maintainer name:
+- Create the new package with the specified maintainer name:
```
Comment (navarroaxel) [inline pages/windows/choco-new.md:14]:
```suggestion
- Create a new package with the specified maintainer name:
```

File: pages/windows/choco-new.md [hunk 12]
```diff
@@ -1,24 +1,24 @@
...
-> More information: <https://chocolatey.org/docs/commands-new>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/new>.
 
-- Create a new package skeleton:
+- Create the new package skeleton:
 
 `choco new {{package_name}}`
 
-- Create a new package with a specific version:
+- Create the new package with the specified version:
```
Comment (navarroaxel) [inline pages/windows/choco-new.md:10]:
```suggestion
- Create a new package with the specified version:
```

File: pages/windows/choco-new.md (hunk 1 above)
Comment (navarroaxel) [reply pages/windows/choco-new.md:6]:
```suggestion
- Create a new package skeleton:
```

File: pages/windows/choco-outdated.md [hunk 13]
```diff
@@ -1,20 +1,20 @@
...
 `choco outdated --ignore-pinned`
 
-- Specify a custom source to check packages from:
+- Check packages from the specified source:
 
-`choco outdated --source {{source_url|alias}}`
+`choco outdated --source {{source_url|source_alias}}`
 
-- Provide a username and password for authentication:
+- Provide the username and the password for authentication:
```
Comment (navarroaxel) [inline pages/windows/choco-outdated.md:18]:
```suggestion
- Provide a specific username and password for authentication:
```

File: pages/windows/choco-pack.md [hunk 14]
```diff
@@ -1,16 +1,16 @@
 # choco pack
 
 > Package a NuGet specification into a nupkg file.
-> More information: <https://chocolatey.org/docs/commands-pack>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/pack>.
 
-- Package a NuGet specification to a nupkg file:
+- Package the NuGet specification to the specified nupkg file:
```
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/windows/choco-pack.md:6]:
```suggestion
- Package a NuGet specification to a specified nupkg file:
```

File: pages/windows/choco-pack.md [hunk 15]
```diff
@@ -1,16 +1,16 @@
...
 `choco pack {{path/to/specification}}`
 
-- Package a NuGet specification specifying the version of the resulting file:
+- Package the NuGet specification specifying the version of the resulting file:
 
-`choco pack {{path/to/specification}} --version {{version}}`
+`choco pack {{path/to/specification}} --version {{specification_version}}`
 
-- Package a NuGet specification to a specific directory:
+- Package the NuGet specification to the specified directory:
```
Comment (navarroaxel) [inline pages/windows/choco-pack.md:14]:
```suggestion
- Package a NuGet specification to a specified directory:
```

File: pages/windows/choco-pack.md [hunk 16]
```diff
@@ -1,16 +1,16 @@
...
-> More information: <https://chocolatey.org/docs/commands-pack>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/pack>.
 
-- Package a NuGet specification to a nupkg file:
+- Package the NuGet specification to the specified nupkg file:
 
 `choco pack {{path/to/specification}}`
 
-- Package a NuGet specification specifying the version of the resulting file:
+- Package the NuGet specification specifying the version of the resulting file:
```
Comment (navarroaxel) [inline pages/windows/choco-pack.md:10]:
```suggestion
- Package a NuGet specification specifying the version of the resulting file:
```

File: pages/windows/choco-pin.md [hunk 17]
```diff
@@ -2,20 +2,20 @@
...
+> More information: <https://docs.chocolatey.org/en-us/choco/commands/pin>.
 
-- Display a list of pinned packages and their versions:
+- Print all pinned packages and their versions:
 
-`choco pin list`
+`choco pin`
 
-- Pin a package at its current version:
+- Pin the specified package at its current version:
```
Comment (navarroaxel) [inline pages/windows/choco-pin.md:11]:
```suggestion
- Pin a specified package at its current version:
```

File: pages/windows/choco-pin.md [hunk 18]
```diff
@@ -2,20 +2,20 @@
...
 
-`choco pin add --name {{package}}`
+`choco pin add --name {{package_name}}`
 
-- Pin a package at a specific version:
+- Pin the package at the specified version:
```
Comment (navarroaxel) [inline pages/windows/choco-pin.md:15]:
```suggestion
- Pin a package at a specified version:
```

File: pages/windows/choco-pin.md [hunk 19]
```diff
@@ -2,20 +2,20 @@
...
 
-`choco pin add --name {{package}} --version {{version}}`
+`choco pin add --name {{package_name}} --version {{package_version}}`
 
-- Remove a pin for a specific package:
+- Remove the pin for the specified package:
```
Comment (navarroaxel) [inline pages/windows/choco-pin.md:19]:
```suggestion
- Remove a pin for a specified package:
```

File: pages/windows/choco-apikey.md [hunk 20]
```diff
@@ -1,20 +1,20 @@
...
-> More information: <https://chocolatey.org/docs/commands-apikey>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/api-key>.
 
-- Display a list of sources and their API keys:
+- Print all sources and their API keys:
 
 `choco apikey`
 
-- Display a specific source and its API key:
+- Print the specified source and its API key:
```
Comment (navarroaxel) [inline pages/windows/choco-apikey.md:10]: We try to avoid `its`
```suggestion
- Print a specified source and the associated API key:
```

**Comment by EmilySeville7cfg** [comment]
> Why do you want to replace almost all "a" with "the"?

I thought that `the` sounds better. But @navarroaxel told be that in Tl;Dr pages we have to prefer `a`/`an` articles to make descriptions more general. It's just my mistake.

File: pages/windows/assoc.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/windows/assoc.md:17]: It's a valid syntax according to [docs][docs]:

> To remove the file type association for a file name extension, add a white space after the equal sign by pressing the SPACEBAR.

[docs]: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/assoc#remarks

File: pages/windows/choco-apikey.md (hunk 20 above)
Comment (EmilySeville7cfg) [reply pages/windows/choco-apikey.md:10]: Why?

Comment (navarroaxel) [reply pages/windows/choco-apikey.md]: I assume possible issues with your, his, its, etc

**Pull request closed without merging**